### PR TITLE
test: add jaeger and gw tests to opi-test

### PR DESCRIPTION
**Pull request by artek-koltun**

### CODE REVIEW

**Review by glimchb — APPROVED** [review]
Lgtm

File: docker-compose.yml [hunk 1]
```diff
@@ -68,15 +68,19 @@ services:
...
+      opi-gw-test:
+        condition: service_completed_successfully # to get at least one operation registered in jaeger
     command: sh -c 'curl -s "http://jaeger:16686/api/traces?service=opi-intel-bridge&lookback=20m&prettyPrint=true&limit=10" | grep operationName'
 
   opi-test:
     image: docker.io/namely/grpc-cli
     networks:
       - opi
     depends_on:
-      opi-intel-server:
```
Comment (glimchb) [inline docker-compose.yml:78]: need to keep this dep..

Comment (artek-koltun) [reply docker-compose.yml:78]: I think `opi-test` should depend on only other tests. And if those tests need `opi-intel-server`, they should specify it explicitly in their `depends_on` section.

Comment (glimchb) [reply docker-compose.yml:78]: maybe... wasn't clear from this commit what your intentions are...

Comment (artek-koltun) [reply docker-compose.yml:78]: well... I can get it back for other PRs, if you insist and remove them in separate PRs

Comment (glimchb) [reply docker-compose.yml:78]: no need, explanation is clear

**Pull request merged**